### PR TITLE
return messages to cspy

### PR DESCRIPTION
**Pull request by c-poole**
Adds a method to tcp class that formats and sends whatever message LabVIEW needs to CsPy back to CsPy.

### CODE REVIEW

**Review by prhuft — APPROVED** [review]
Looks good.

File: hamamatsu.py [hunk 1]
```diff
@@ -552,21 +553,21 @@ def data_out(self) -> str:
...
-        hm_str += format_data(f"{hm}/columns", f"{sz[2]}")
+        hm_str += TCP.format_data(f"{hm}/numShots", f"{sz[0]}")
+        hm_str += TCP.format_data(f"{hm}/rows", f"{sz[1]}")
+        hm_str += TCP.format_data(f"{hm}/columns", f"{sz[2]}")
 
         for shot in range(sz[0]):
             flat_ar = np.reshape(self.last_measurement[shot, :, :], sz[1]*sz[2])
-            tmp_str = ar_to_str(flat_ar)  # TODO : implement based on encoding in cspy and labview
-            hm_str += format_data(f"{hm}/shots/{shot}", tmp_str)
+            tmp_str = u16_ar_to_str(flat_ar)
```
Comment (JuanBohorquez3) [inline hamamatsu.py:564]: Further up it's not assumed the array will be u16 (u8 and u32 are options) but for the Hamamatsu I'm pretty sure the data is in u16. 
If you can make the u8 and u32 methods I can set up the logic checking so the conversion goes smoothly in the general case.

Comment (c-poole) [reply hamamatsu.py:564]: Was the logic that auto-forced images to u16 not preserved?

Comment (c-poole) [reply hamamatsu.py:564]: I should note that I chose this because CsPy assumes u16. CsPy should fundamentally fail if it received another data type.

Comment (JuanBohorquez3) [reply hamamatsu.py:564]: Oh jeez. Ok I'll add a little error message if the data format isn't u16 in that case

Comment (c-poole) [reply hamamatsu.py:564]: An error message isn't necessary. Simply casting the array to u16 once the data is taken during the measurement will suffice.

Comment (c-poole) [reply hamamatsu.py:564]: And that is the strategy currently implemented by LabVIEW

Comment (JuanBohorquez3) [reply hamamatsu.py:564]: Ok, taking a look, the data array is currently filled with python ints so the casting should be the same in all cases

Comment (JuanBohorquez3) [reply hamamatsu.py:564]: Does your u16 function perform the casting or should that be done explicitly?

**Review by JuanBohorquez3 — APPROVED** [review]
All's good

**Pull request merged**